scripts/hub_bootstrap.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install python3-venv python3-dev python3-setuptools
+
+python3 -m venv hub
+cd hub
+source bin/activate
+pip install --upgrade pip
+python3 -m pip install wheel
+python3 -m pip install homeassistant
+cd ..
+
+export CURRENT_USER=`whoami`
+cat home-assistant.service | sed -e s/homeassistant/$CURRENT_USER/g > home-assistant.service
```

Comment:
There is no need in that as you can use literal percent parameter (`%i`/`%I`) for `execstart` inside of systemctl service template.
e.g.
`ExecStart=/bin/bash -c '/home/%i/hub/bin/hass -c "/home/%i/.hub"'`

Reply:
that is too broad, I want to be more specific